### DISCUSSION TITLE
Profile location

### DISCUSSION THREAD

**Discussion by dragomano** [goastian/midori-desktop#43] [Q&A]
Why portable version creates and uses profile in `C:\Users\User\AppData\Roaming\Midori\Profiles\bla-bla-bla.default-release`? Why not in the app dir?

**Comment by Pyromane** [comment]
Ping @Ponchale

**Comment by Pyromane** [comment]
In the meantime, I have built the following two bat files as a workaround:

backup.bat
```
ROBOCOPY %appdata%\Midori %CD%\ProfilBackup /MIR

pause
```

restore.bat
```
ROBOCOPY %CD%\ProfilBackup %appdata%\Midori /MIR

pause
```

**Reply by Ponchale** [reply]
we work for fix this

**Comment by ioc2e3** [comment]
Let ( portable version's personal data ) be on ( Portable version's program directory ) ...

... Because , not everyone knows how to use bat files

.

**Comment by Pyromane** [comment]
Why is this marked as closed if the problem still exists?
